CHANGELOG.md

```diff
@@ -1,6 +1,24 @@
 Change Log
 ==========
 
+[v0.5.0-c1][] (2017-07-24)
+--------------------------
+
+bug fixes and improvements, make it more stable.
+
+- Bugs
+    + fixed the problem on windows with chocolate and bash-completion.
+    + upgraded and pin down with newest chef version.
+    + fixed problem with rsync_args (thanks to @smxsm).
+
+- Improvements
```

Comment:
https://github.com/teracyhq/dev/issues?q=is%3Aissue+label%3Atype%3Afeature+milestone%3Av0.5.0-c1-dev+is%3Aclosed